###### Respondendo hipóteses

Neste notebook iremos realizar as consultas de nossas tabela gold e responder, de fato, as hipóteses/perguntas de negócio no trabalho inicial.

In [0]:
%sql
SELECT *
FROM mvp_bernardomoraes_catalog.gold.causas_acidentes_fatalidades
ORDER BY total_fatalidades DESC
LIMIT 10;

fator_acidente,area_fator,qtd_acidentes,total_fatalidades
null,null,1074,475
APLICAÇÃO DE COMANDOS,FATOR OPERACIONAL,317,169
ATITUDE,FATOR HUMANO,83,92
ATENÇÃO,FATOR HUMANO,49,36
ANSIEDADE,FATOR HUMANO,4,23
CONDIÇÕES METEOROLÓGICAS ADVERSAS,FATOR OPERACIONAL,31,23
CAPACITAÇÃO E TREINAMENTO,FATOR HUMANO,10,14
INDISCIPLINA DE VOO,FATOR OPERACIONAL,41,11
CARACTERÍSTICAS DA TAREFA,FATOR HUMANO,3,10
MANUTENÇÃO DA AERONAVE,FATOR OPERACIONAL,84,8


In [0]:
%sql
SELECT fabricante_aeronave, modelo_aeronave, SUM(qtd_ocorrencias) AS total
FROM mvp_bernardomoraes_catalog.gold.fabricante_modelo_falhas_tecnicas
GROUP BY fabricante_aeronave, modelo_aeronave
ORDER BY total DESC
LIMIT 10;

fabricante_aeronave,modelo_aeronave,total
AEROSPATIALE AND ALENIA,ATR-72-212A,127
EMBRAER,ERJ 190-200 IGW,102
CESSNA AIRCRAFT,152,92
EMBRAER,EMB-810C,79
NEIVA INDUSTRIA AERONAUTICA,EMB-810D,79
null,null,69
BEECH AIRCRAFT,58,56
CESSNA AIRCRAFT,208B,54
PIPER AIRCRAFT,PA-34-220T,35
EMBRAER,EMB-810D,35


In [0]:
%sql
SELECT aerodromo_ocorrencia, SUM(qtd_ocorrencias) AS total
FROM mvp_bernardomoraes_catalog.gold.aerodromo_fator_humano
GROUP BY aerodromo_ocorrencia
ORDER BY total DESC
LIMIT 10;

aerodromo_ocorrencia,total
SBBR,13
SBBP,11
SBMT,11
SBBI,10
SBLO,10
SBMG,9
SBNV,9
SBGO,9
SWFN,8
SBPR,7


##### 1. `Quais causas de acidentes geram maior gravidade (com maior número de feridos)? Para assim conseguirmos direcionar quais causas devemos investigar e tentar ao máximo mitigar.`

| Causa | Área do fator | Qtd. acidentes | Fatalidades |
|---|---|---|---|
| APLICAÇÃO DE COMANDOS | FATOR OPERACIONAL | 317 | 169 |
| ATITUDE | FATOR HUMANO | 83 | 92 |
| ATENÇÃO | FATOR HUMANO | 49 | 36 |
| CONDIÇÕES METEOROLÓGICAS ADVERSAS | FATOR OPERACIONAL | 31 | 23 |
| MANUTENÇÃO DA AERONAVE | FATOR OPERACIONAL | 84 | 8 |

✅ Com a análise original a categoria `APLICAÇÃO DE COMANDOS` segue como maior causa em fatalidades (169), com os mesmos 317 acidentes, além de `CONDIÇÕES METEOROLÓGICAS ADVERSAS` e  `MANUTENÇÃO DA AERONAVE` serem fatores operacionacionais relevantes a serem investigados.

Vale também o entendimento um pouco maior do TOP 2 e TOP 3 causas, que são referentes a fatores humanos, entendendo a possibilidade de mitigar isso através de treinamentos ou ações das lideranças em relação as equipes.

⚠️ **Observação:** a primeira linha (contendo registros de 1.074 acidentes, 475 fatalidades), são nulas, ou seja, representando os acidentes **sem investigação publicada**. Dessa forma, esses registros não são erros,mas sim o gap de cobertura que já sabíamos existir desde o início (~81% da base não tem `fator_acidente` preenchido). Vale manter essa linha visível na gold para não esconder esse gap, mas é importante deixar claro em qualquer dashboard ou anaálise que ela não é "uma causa", e sim "causa não identificada/investigada".

##### 2. `Quais os fabricantes ou modelos de aeronaves com maior incidencia de acidentes relacionadas a questões técnicas/de infraestrutura do avião.`

| Fabricante | Modelo | Total |
|---|---|---|
| AEROSPATIALE AND ALENIA | ATR-72-212A | 127 |
| EMBRAER | ERJ 190-200 IGW | 102 |
| CESSNA AIRCRAFT | 152 | 92 |
| EMBRAER | EMB-810C | 79 |
| NEIVA INDUSTRIA AERONAUTICA | EMB-810D | 79 |

✅ Com a análise original, o ATR-72-212A da Aerospatiale/Alenia segue na liderança (127 vs. 136 originais; pequena diferença explicada pelo tratamento de fabricante/nulos aplicado na silver).

⚠️ **Observação:** linhas nulas (69 ocorrências) indica registros sem fabricante/modelo identificado na fonte, mesmo raciocínio do ponto anterior, vale sinalizar no dashboard ou análises pontuais.

##### 3. `Quais aeroportos tem maiores incidentes relacionados a funcionários/erros humanos? Com intuito de priorizarmos treinamentos nesses locais.`

| Aeródromo | Total |
|---|---|
| SBBR (Brasília) | 13 |
| SBBP (Bragança Paulista) | 11 |
| SBMT (Campo de Marte) | 11 |
| SBBI (Bacacheri) | 10 |
| SBLO (Londrina) | 10 |

✅ Com a análise original, SBBR segue na liderança com 13 ocorrências.
